js/classes/entities/midground/chad.js

```diff
@@ -168,15 +180,16 @@ class Chad {
      */
     takeDamage(amount) {
         if (this.isInvincible) {
```

Comment:
So if Chad is invisible then nothing happens other than playing a sound? Or is there functionality somewhere else?